packages/commands/src/purchase-content.mjs

```diff
@@ -79,3 +80,39 @@ program
 
     process.exit(0)
   })
+
+
+program.command('purchase-track')
```

Comment:
why not just use purchase-content above? can we keep one or the other?

Reply:
Didn't want to do albums yet